_includes/disclaimer.html

```diff
@@ -1,5 +1,12 @@
-<div class="disclaimer">
+<header class="site-header">
```

Comment:
B/c `<header>` may exist on other pages we are building and needed to add some sort of differentiation to this one.